core-spi-test/src/main/java/org/ehcache/internal/store/StoreFactory.java

```diff
@@ -51,6 +52,8 @@
 
   ServiceConfiguration<?>[] getServiceConfigurations();
 
+  ServiceProvider getServiceProvider();
```

Comment:
Minor detail, but this feels it shouldn't be there. More so all implementations are actually _instantiating_ a new `ServiceLocator` on invocation. I'd rather add the ServiceProvider to the `SPIStoreTester` constructor, just like the `StoreFactory` instance is injected in the tests